### PR TITLE
[BOLT][DOC] Add script for automatic user guide generation

### PR DESCRIPTION
**Pull request by ElvinaYakubova**

### CODE REVIEW

**Review by rafaelauler — APPROVED** [review]
Thanks!

File: bolt/lib/Rewrite/DWARFRewriter.cpp [hunk 1]
```diff
@@ -352,7 +352,7 @@ static cl::opt<bool> CreateDebugNames(
 
 static cl::opt<bool>
     DebugSkeletonCu("debug-skeleton-cu",
-                    cl::desc("prints out offsetrs for abbrev and debu_info of "
+                    cl::desc("prints out offsets for abbrev and debug_info of "
```
Comment (maksfb) [inline bolt/lib/Rewrite/DWARFRewriter.cpp:355]: nit: it's better to put this change under a separate "[NFC]" commit.

**Review by maksfb — COMMENTED** [review]
Thanks a lot!

File: bolt/lib/Rewrite/DWARFRewriter.cpp (hunk 1 above)
Comment (ElvinaYakubova) [reply bolt/lib/Rewrite/DWARFRewriter.cpp]: thanks! done in [PR#93955](https://github.com/llvm/llvm-project/pull/93955)

**Pull request merged**